### DISCUSSION TITLE
Add `--version` option for Brew package

### DISCUSSION THREAD

**Discussion by pervoj** [Prince781/vala-language-server#209] [Ideas]
I am working on building VLS for Brew. However, I don't know how to write tests for VLS. Is there any parameter which VLS takes? And any output for it?

I see there is `vls-testclient` executable, but how to run it, when `install` is set to `false`?

My current configuration file for Brew:

```rb
class ValaLanguageServer < Formula
  desc "Code Intelligence for Vala & Genie"
  homepage "https://github.com/Prince781/vala-language-server"
  url "https://github.com/Prince781/vala-language-server/archive/refs/tags/0.48.3.tar.gz"
  sha256 "1a752515efda561aec535d088a6f52888b13ad2d52f2d27770ddf6a519f0028b"
  license "LGPL-2.1"

  depends_on "meson" => :build
  depends_on "pkg-config" => :build
  depends_on "vala" => :build
  depends_on "glib"
  depends_on "json-glib"
  depends_on "jsonrpc-glib"
  depends_on "libgee"

  def install
    system "meson", "-Dplugins=false", "-Dprefix=#{prefix}", "build"
    system "ninja", "-C", "build"
    system "ninja", "-C", "build", "install"
  end

  test do
    # test will be here
  end
end
```

Everything works great, the tests are the only problem.

**Comment by Prince781** [comment]
There are currently no tests for VLS. (See #26).

**Reply by pervoj** [reply]
So, can you please release a new version with the `--version` parameter? Because checking the version dump is an acceptable way of testing, according to Brew.

**Reply by Prince781** [reply]
Done. See 3e01b83

**Comment by Prince781** [comment]
Also beware that VLS installs its GNOME Builder plugin, so you need to include `"-Dplugins=false"` in the first meson command.

**Reply by pervoj** [reply]
Done

**Comment by pervoj** [comment]
Done and waiting for merging, see Homebrew/homebrew-core#80981

**Reply by Prince781** [reply]
Seeing as the homebrew package was merged, we can say this topic has been resolved.
